controller/src/test/java/io/pravega/controller/timeout/TimeoutServiceTest.java

```diff
@@ -319,29 +319,14 @@ public void testPingFailureMaxExecutionTimeExceeded() throws InterruptedExceptio
         timeoutService.addTxn(SCOPE, STREAM, txData.getId(), txData.getVersion(), LEASE,
                 txData.getMaxExecutionExpiryTime(), txData.getScaleGracePeriod());
 
-        Optional<Throwable> result = timeoutService.getTaskCompletionQueue().poll((long) (0.75 * LEASE), TimeUnit.MILLISECONDS);
-        Assert.assertNull(result);
-
         TxnStatus status = streamStore.transactionStatus(SCOPE, STREAM, txData.getId(), null, executor).join();
         Assert.assertEquals(TxnStatus.OPEN, status);
 
-        PingTxnStatus pingStatus = timeoutService.pingTxn(SCOPE, STREAM, txData.getId(), LEASE);
-        Assert.assertEquals(PingTxnStatus.Status.OK, pingStatus.getStatus());
-
-        result = timeoutService.getTaskCompletionQueue().poll((long) (0.75 * LEASE), TimeUnit.MILLISECONDS);
-        Assert.assertNull(result);
-
-        status = streamStore.transactionStatus(SCOPE, STREAM, txData.getId(), null, executor).join();
-        Assert.assertEquals(TxnStatus.OPEN, status);
-
-        pingStatus = timeoutService.pingTxn(SCOPE, STREAM, txData.getId(), LEASE + 1);
+        PingTxnStatus pingStatus = timeoutService.pingTxn(SCOPE, STREAM, txData.getId(), 3 * LEASE);
```

Comment:
I guess we want a leave value > MAX_EXECUTION_TIME here. can make this explicit by using MAX_EXECUTION_TIME or comment that 3 * LEASE is > MAX_EXECUTION_TIME